hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -229,53 +238,70 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     super(props);
     this.state = {
       loading: false,
-      missingDataSource: [],
-      underReplicatedDataSource: [],
-      overReplicatedDataSource: [],
-      misReplicatedDataSource: [],
-      expandedRowData: {}
+      containerDataSource: [],
+      expandedRowData: {},
+      missingCount: 0,
+      underReplicatedCount: 0,
+      overReplicatedCount: 0,
+      misReplicatedCount: 0,
+      DEFAULT_LIMIT: 10,
+      currentState: "MISSING",
+      firstSeenKey: '0',
+      lastSeenKey: '0',
+      currentPage: 1
     };
   }
 
-  componentDidMount(): void {
-    // Fetch missing containers on component mount
+  fetchUnhealthyContainers = (direction: boolean) => {
     this.setState({
+      currentPage: 1,
       loading: true
     });
-
-    const { request, controller } = AxiosGetHelper('/api/v1/containers/unhealthy', cancelContainerSignal);
+    let baseUrl =
+        `/api/v1/containers/unhealthy/${encodeURIComponent(this.state.currentState)}?limit=${total_fetch_entries}`;
```

Comment:
`total_fetch_entries` is not required, as if we fetch 1000 records by default AntD will handle the pagination by default.
We should fetch only the `pageSize` amount and paginate further on the basis of minContainerId and maxContainerId
```suggestion
        `/api/v1/containers/unhealthy/${encodeURIComponent(this.state.currentState)}?limit=${this.state.pageSize}`;
```